etcdserver/cluster_util.go

```diff
@@ -229,7 +319,7 @@ func decideDowngradeStatus(lg *zap.Logger, targetVersion *semver.Version, vers m
 		if ver == nil {
 			return false
 		}
-		v, err := semver.NewVersion(ver.Server)
+		v, err := semver.NewVersion(version.Cluster(ver.Server) + ".0")
 		if err != nil {
```

Comment:
Why do we need .0? Is that for patch release?

Reply:
It is to avoid the format error of `semver.NewVersion`. I think the patch version should be ignored for downgrade.

Reply:
Right, that's why I am asking why we want to add that .0, as long as downgrade ignore patch version, that's fine.